lib/bullet_train/billing/engine.rb

```diff
@@ -1,6 +1,25 @@
 module BulletTrain
   module Billing
     class Engine < ::Rails::Engine
+      initializer "bullet_train-billing.quantity" do
+        ActiveSupport::Notifications.subscribe("memberships.quantity-changed") do |_name, _start, _finish, _id, payload|
+          billing_subscription = payload[:team].current_billing_subscription
+
+          next unless billing_subscription.present?
+
+          included_price = billing_subscription.included_prices.first
+
+          # if not billing per seat, return
+          next unless included_price.price[:quantity] == "memberships"
+
+          new_quantity = included_price.price.calculate_quantity(payload[:team])
+
+          # return if quantity hasn't changed
+          next if included_price[:quantity] == new_quantity
+
+          ActiveSupport::Notifications.instrument("memberships.provider-quantity-changed", {provider_subscription: billing_subscription.provider_subscription, included_price:, quantity: new_quantity})
```

Comment:
`provider-quantity-changed` is a little ambiguous as to what it means. I think it would be clearer as `provider-subscription-quantity-changed`.

```suggestion
          ActiveSupport::Notifications.instrument("memberships.provider-subscription-quantity-changed", {provider_subscription: billing_subscription.provider_subscription, included_price:, quantity: new_quantity})
```

Reply:
Works for me 👍🏻